migrator/migrator_test.go

```diff
@@ -123,6 +123,141 @@ func TestRunStrategiesInOrder(t *testing.T) {
 	assert.Equal(t, []int{3, 1, 2}, result)
 }
 
+// TestNodePortStrategyIntegration tests the integration of the NodePort strategy with the migrator
+func TestNodePortStrategyIntegration(t *testing.T) {
```

Comment:
I think this should be renamed to use some other word than `integration`, so that we don't wrongly assume it is an integration test.